judo-runtime-core-jsl-itest/src/test/resources/model/Functions.jsl

```diff
@@ -298,11 +298,61 @@ entity TimestampFunctions {
+    field Timestamp timestamp12 = `2022-03-03T14:00:00.123+03:03`;
+    field Timestamp timestamp13 = `2022-03-03T14:00:00.123-00:03`;
+    field Timestamp timestamp14 = `2022-03-03T14:00:00.123-03:00`;
+    field Timestamp timestamp15 = `2022-03-03T14:00:00.123-03:03`;
+    field Timestamp timestamp16 = `2022-03-03T14:00:00.123Z`;
+    field Timestamp timestamp17 = `2022-03-03T14:00:00Z`;
+
+    derived String timestamp1AsString => `2019-07-18T01:11:12+02:00`!asString();
+    derived String timestamp1AsString1 => self.timestamp1!asString();
+    derived String timestamp2AsString => `2019-07-18T01:11:12Z`!asString();
+    derived String timestamp2AsString1 => self.timestamp2!asString();
+    derived String timestamp3AsString => `2022-03-03T14:00:00+00:03`!asString();
+    derived String timestamp3AsString1 => self.timestamp3!asString();
+    derived String timestamp4AsString => `2022-03-03T14:00:00+03:00`!asString();
+    derived String timestamp4AsString1 => self.timestamp4!asString();
+    derived String timestamp5AsString => `2022-03-03T14:00:00+03:03`!asString();
+    derived String timestamp5AsString1 => self.timestamp5!asString();
+    derived String timestamp6AsString => `2022-03-03T14:00:00-00:03`!asString();
+    derived String timestamp6AsString1 => self.timestamp6!asString();
+    derived String timestamp7AsString => `2022-03-03T14:00:00-03:00`!asString();
+    derived String timestamp7AsString1 => self.timestamp7!asString();
+    derived String timestamp8AsString => `2022-03-03T14:00:00-03:03`!asString();
+    derived String timestamp8AsString1 => self.timestamp8!asString();
+    derived String timestamp9AsString => `2022-03-03T14:00:00.123+00:03`!asString();
+    derived String timestamp9AsString1 => self.timestamp9!asString();
+    derived String timestamp10AsString => `2022-03-03T14:00:00.123+03:00`!asString();
+    derived String timestamp10AsString1 => self.timestamp10!asString();
+    derived String timestamp12AsString => `2022-03-03T14:00:00.123+03:03`!asString();
+    derived String timestamp12AsString1 => self.timestamp12!asString();
+    derived String timestamp13AsString => `2022-03-03T14:00:00.123-00:03`!asString();
+    derived String timestamp13AsString1 => self.timestamp13!asString();
+    derived String timestamp14AsString => `2022-03-03T14:00:00.123-03:00`!asString();
+    derived String timestamp14AsString1 => self.timestamp14!asString();
+    derived String timestamp15AsString => `2022-03-03T14:00:00.123-03:03`!asString();
+    derived String timestamp15AsString1 => self.timestamp15!asString();
+    derived String timestamp16AsString => `2022-03-03T14:00:00.123Z`!asString();
+    derived String timestamp16AsString1 => self.timestamp16!asString();
+    derived String timestamp17AsString => `2022-03-03T14:00:00Z`!asString();
+    derived String timestamp17AsString1 => self.timestamp17!asString();
+
     derived String ownTimestamp1AsString => self.timestamp1!asString();
```

Comment:
The ownTimestamp1AsString and ownTimestamp2AsString fields don't use anywhere in the FunctionsTest.java.

Reply:
`ownTimestamps` removed